bot/utils/scheduling.py

```diff
@@ -84,32 +115,29 @@ def _task_done_callback(self, task_id: t.Hashable, done_task: asyncio.Task) -> N
         If `done_task` and the task associated with `task_id` are different, then the latter
         will not be deleted. In this case, a new task was likely rescheduled with the same ID.
         """
-        log.trace(f"{self.cog_name}: performing done callback for task #{task_id} {id(done_task)}.")
+        self._log.trace(f"Performing done callback for task #{task_id} {id(done_task)}.")
 
         scheduled_task = self._scheduled_tasks.get(task_id)
 
         if scheduled_task and done_task is scheduled_task:
-            # A task for the ID exists and its the same as the done task.
+            # A task for the ID exists and is the same as the done task.
             # Since this is the done callback, the task is already done so no need to cancel it.
-            log.trace(f"{self.cog_name}: deleting task #{task_id} {id(done_task)}.")
+            self._log.trace(f"Deleting task #{task_id} {id(done_task)}.")
             del self._scheduled_tasks[task_id]
         elif scheduled_task:
             # A new task was likely rescheduled with the same ID.
-            log.debug(
-                f"{self.cog_name}: the scheduled task #{task_id} {id(scheduled_task)} "
+            self._log.debug(
+                f"The scheduled task #{task_id} {id(scheduled_task)} "
                 f"and the done task {id(done_task)} differ."
             )
         elif not done_task.cancelled():
-            log.warning(
-                f"{self.cog_name}: task #{task_id} not found while handling task {id(done_task)}! "
+            self._log.warning(
+                f"Task #{task_id} not found while handling task {id(done_task)}! "
                 f"A task somehow got unscheduled improperly (i.e. deleted but not cancelled)."
             )
 
         with contextlib.suppress(asyncio.CancelledError):
             exception = done_task.exception()
             # Log the exception if one exists.
             if exception:
```

Comment:
`CancelledError` will not be returned, it will be raised, right? 

> If the Task has been cancelled, this method raises a CancelledError exception.

Reply:
Yes, but I was under the impression that with the `contextlib.suppress(asyncio.CancelledError):`, it would instead return the `CancelledError` from `exception = done_task.exception()` if the task was cancelled instead of raising like it normally would. I just confirmed that was not the case, I forgot that it if the exception occurs under `contextlib.suppress()`, it continues *after* the block (rather than at the point the exception occurred). So, feel free to disregard the above suggestion.

(I haven't personally used `contextlib.suppress()` much at all in my own code.)